src/components/noteOverlay/noteOverlay.jsx

```diff
@@ -37,14 +40,20 @@ function NoteOverlay(){
             <textarea name="body" className="note-body f-s" value={note.body}></textarea>
             <div className="note-footer end">
               <h5>Priority:</h5>
-              <input type="radio" name="priority" value='High'></input>
+              <input type="radio" name="priority" value={1}></input>
               <label>High</label>
-              <input type="radio" name="priority" value='Medium'></input>
+              <input type="radio" name="priority" value={2}></input>
               <label>Medium</label>
-              <input type="radio" name="priority" value='Low'></input>
+              <input type="radio" name="priority" value={3}></input>
               <label>Low</label>
                 <FontAwesomeIcon onClick={()=>changeCardColor(note.color)} icon={faPalette}></FontAwesomeIcon>
-                <FontAwesomeIcon icon={faTag}></FontAwesomeIcon>
+                <FontAwesomeIcon icon={faTag} onClick={()=>setShowTags("flex")}></FontAwesomeIcon>
+            </div>
+            <div style={{display:`${showTags}`}}>
+              {state.labels.map((item)=><div className="tag-menu"> <input type="checkbox" name="tags" value={item.tag}></input>
+            <label className="badge badge-text primary">{item.tag}</label>
+              </div>
+              )}
             </div>
           </div>
           <button className="btn primary-bg f-m center-txt" type="submit" >Submit</button>
```

Comment:
Create or Add Note would make more sense than Submit here.

Reply:
Agreed, If it's Add or Create a note, then it would be a better User experience.